### PR TITLE
Lib-utc

### PR DESCRIPTION
**Pull request by milliorn**

### CODE REVIEW

File: src/utc.rs [hunk 1]
```diff
@@ -8,236 +8,428 @@ use super::*;
...
     pub wings_new: Option<NwValue<i32>>,
+
+    // The wisdom attribute value of the creature.
     #[serde(skip_serializing_if = "Option::is_none", rename = "Wis")]
     pub wisdom: Option<NwValue<i32>>,
 }
 
 #[derive(Debug, Serialize, Deserialize)]
 pub struct Class {
+    /// Identifier for the struct (optional)
```
Comment (urothis) [inline src/utc.rs]: Don't think I like this (optional) bit, the type implies that. No need to repeat ourselves.

File: src/lib.rs [hunk 2]
```diff
@@ -129,22 +130,27 @@ impl NwType {
...
+/// Represents a named value in the Neverwinter Nights module file format.
 #[derive(Debug, Serialize, Deserialize)]
 pub struct NwValue<T> {
     #[serde(rename = "type")]
-    pub _type: String,
+    pub _type: String, // Represents the type of the value
+
     #[serde(rename = "value")]
-    pub value: T,
+    pub value: T, // The actual value of type T
```
Comment (urothis) [inline src/lib.rs]: What does this look like in doc.rs?

File: src/utc.rs (hunk 1 above)
Comment (milliorn) [reply src/utc.rs]: (optional) removed in commit https://github.com/milliorn/rs_nwn_nasher_types/pull/9/commits/e5f165459f4bb4a96293e4258ede07815769d18e

File: src/lib.rs (hunk 2 above)
Comment (milliorn) [reply src/lib.rs]: It looks like someone doesn't know what they are doing. Fixing it.

Comment (milliorn) [reply src/lib.rs]: https://github.com/milliorn/rs_nwn_nasher_types/pull/9/commits/3fd4195b0b345d8a0a031b908ca6785847b26c4b

**Review by urothis — APPROVED** [review]

**Pull request merged**